widget_driver_test/lib/src/utils/test_drivable_widget.dart

```diff
@@ -12,10 +12,9 @@ class TestDrivableWidget<T extends WidgetDriver> extends DrivableWidget<T> {
   final T Function(BuildContext context) _driverBuilder;
 
   TestDrivableWidget({
-    Key? key,
     required T Function(BuildContext context) driverBuilder,
   })  : _driverBuilder = driverBuilder,
-        super(key: key, environmentInfo: TestRuntimeEnvironmentInfo());
+        super(key: UniqueKey(), environmentInfo: TestRuntimeEnvironmentInfo());
```

Comment:
Hmm, it might make more sense to pass in the unique key.
When we define it here in the inside on this widget we basically kill the reuse-logic of Flutter.
So in the future if we would ever create this widget inside a build method, then we would get a bug.

Maybe change this and pass in the unique key from test test-helper which creates this widget

Reply:
Done :)